### PR TITLE
Name parameter introduced in Organization Search

### PR DESCRIPTION
**Pull request by NimishaMN**
Description
----
Name parameter introduced in Organization Search

Screenshot 2021-03-02 at 3 51 57 PM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/entities/organization.md [hunk 1]
```diff
@@ -36,12 +36,13 @@ Search for Organizations that meet supplied query parameters:
...
- `type`      | This or `identifier` or `_id`  | [`token`]  | The organization's type. Example: `http://terminology.hl7.org/CodeSystem/organization-type|govt`
- [`_count`]  | No                             | [`number`] | The maximum number of results to return. Defaults to `100`.
+ Name        | Required?                               | Type       | Description
+-------------|-----------------------------------------|------------|----------------------------------------------------------
+ `_id`       | This or `identifier` or `type`          | [`token`]  | The logical resource id associated with the resource.
+ `identifier`| This or `type` or `_id`                 | [`token`]  | NPI, OID or FederalTAXID identifier for the organization.
+ `type`      | This or `identifier` or `_id`           | [`token`]  | The organization's type. Example: `http://terminology.hl7.org/CodeSystem/organization-type|govt`
+ [`_count`]  | No                                      | [`number`] | The maximum number of results to return. Defaults to `100`.
+ `name`      | This or `_id` or `identifier` or `type` | [`string`] | Name used for the organization.
```
Comment (aki-mathur) [inline content/millennium/r4/entities/organization.md:45]: @NimishaMN  Do you have the description mentioned somewhere in jira. As per the [hl7](https://hl7.org/fhir/r4/organization.html) is should be _'A portion of the organization's name or alias'_

**Review by aki-mathur — CHANGES_REQUESTED** [review]
@NimishaMN Can you confirm with SD that do we need to add an example for search by name.

File: content/millennium/r4/entities/organization.md (hunk 1 above)
Comment (NimishaMN) [reply content/millennium/r4/entities/organization.md:45]: Confirmed with SD (Alec) for the Description. Got a +1 that we can go ahead with 'Name used for the organization.'

**Comment by NimishaMN** [comment]
> @NimishaMN Can you confirm with SD that do we need to add an example for search by name.

Confirmed with @ashu-bdvt, that it is not required to add separate example specifically for Search by Name.

**Review by aki-mathur — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Comment by MaxPhilips** [comment]
live
Screen Shot 2021-03-05 at 12 54 22 PM

**Pull request merged**